scripts/create.sh

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# "---------------------------------------------------------"
+# "-                                                       -"
+# "-  Creates cluster and deploys demo application         -"
+# "-                                                       -"
+# "---------------------------------------------------------"
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+# shellcheck disable=SC1090
+source "$ROOT"/scripts/common.sh
+
+# Generate the variables to be used by Terraform
+# shellcheck disable=SC1090
+source "$ROOT"/scripts/generate-tfvars.sh
+
+# Enable required GCP APIs
+gcloud services enable container.googleapis.com
+gcloud services enable cloudtrace.googleapis.com
+
+# Initialize and run Terraform
+(cd "$ROOT"/terraform; terraform init -input=false)
```

Comment:
This puts it in a sub-shell that avoids getting stuck in the $ROOT/terraform directory.

Reply:
bash isn't my strength either, but my understanding is it creates a subshell so that if something goes wrong my context will be in my original directory and not the one I cd-ed into. I'm not sure if this is the best option. I wasn't entirely consistent, either, because there were other situations where I needed to go to a subdir to run a command whose result should be set to a variable and I had to use a different syntax. Would appreciate any suggestions you have.

Reply:
Huh. Didn't see Robin's comment when I commented. Guess I was right, though!